internal/pkg/apprunner/apprunner.go

```diff
@@ -55,6 +57,19 @@ func (c Client) ForceUpdateService(app, env, svc string) error {
 	return c.appRunnerClient.WaitForOperation(id, svcARN)
 }
 
+// LastUpdatedAt returns the last updated time of the app runner service.
+func (c Client) LastUpdatedAt(app, env, svc string) (*time.Time, error) {
```

Comment:
I use the pointer because `time.Time{}` is a valid time so that we can't use `time.Time{}` to denote the field is empty (the same logic as `count`). In terms of safety as long as we use `aws.TimeValue()` I think we should be good? What do you think 🤔

Reply:
Yeah, it makes sense, I think my point was that at least for App Runner this value is always set so it can't ever be `nil`. If that's the case also with ECS we can get rid of an unnecessary condition. Otherwise, we can keep it as is

Reply:
nvm i think they are about the same since `time.Time{}` doesn't really happen